k8s/staging/kube-system/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../../production/kube-system/metrics-server.yaml
```

Comment:
What does this accomplish? Is it just using the same configuration as prod?

If a matching `kustomization.yaml` file is not provided for staging, does it get run on staging?

Reply:
Yup, it's just using the same configuration as prod. If a YAML is not listed in a `kustomization.yaml`, it doesn't get run on staging.

Reply:
The only concern I have is reproducibility between prod and staging, since we're running a subset of the services in staging. However, if you don't see any issue there, then it's fine by me.